src/applications/hca/routes.jsx

```diff
@@ -1,13 +1,33 @@
-import { createRoutesWithSaveInProgress } from '../../platform/forms/save-in-progress/helpers';
+import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
+import environment from 'platform/utilities/environment';
 
 import formConfig from './config/form';
 import HealthCareApp from './HealthCareApp.jsx';
+import IDPage from './containers/IDPage';
 
-const route = {
+const formRoutes = createRoutesWithSaveInProgress(formConfig);
+
+// If the route's path ends in `introduction` then the form system will treat it
+// as an Introduction page (ie it won't add the Title component or navigation
```

Comment:
This feels like something we should make configurable in the forms code.

Reply:
Yeah, I had a PR to configure special routes in the form config but @dmethvin-gov nudged me towards just using a path name that ends in `introduction`: https://github.com/usds/us-forms-system/pull/342

I'm totally open to revisiting that closed PR if you want.

Reply:
I think we should revisit it when we move the code back into vets-website.

Reply:
Agreed. It's frustrating that I couldn't tag people to review that PR I just mentioned because we don't own that repo.